src/ucp/core/ucp_mm.c

```diff
@@ -877,3 +944,97 @@ void ucp_mem_print_info(const char *mem_size, ucp_context_h context, FILE *strea
         printf("<Failed to unmap memory of size %s>\n", mem_size);
     }
 }
+
+ucs_status_t ucp_memh_get(ucp_context_h context, void *address, size_t length,
```

Comment:
can the ucp_memh_get() , when called from data path, be inline all the way to rcache?